src/js/components/Form/stories/ControlledInput.js

```diff
@@ -24,6 +25,7 @@ export const ControlledInput = () => {
   const [size, setSize] = useState('');
   const [comments, setComments] = useState('');
   const [age, setAge] = useState('');
+  const [setFile] = useState('');
```

Comment:
```suggestion
  const [file, setFile] = useState('');
```
and is the default an empty object instead of string? `{}`